pathspider/base.py

```diff
@@ -428,6 +428,7 @@ def merge(self, flow, res):
                             except KeyError:
                                 info = {'dip' : job['dip'], 'hops' : 40}  #'hops' could not be found in flow
                         self.ipqueue.put(info)
+                        print("put in queue")
```

Comment:
Please use the logger functions, this would print into the output file and mess up anything trying to do analysis.